easybuild/easyconfigs/f/fontconfig/fontconfig-2.12.1-foss-2015a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'fontconfig'
+version = '2.12.1'
+
+homepage = 'http://www.freedesktop.org/software/fontconfig'
+description = """Fontconfig is a library designed to provide system-wide font configuration, customization and
+application access."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['http://www.freedesktop.org/software/fontconfig/release/']
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('expat', '2.1.0'),      # Selected an older version for consistency
+    ('freetype', '2.6.2'),   # Selected an older version for consistency
```

Comment:
@valtandor OK to drop the comments imho, it makes sense to use older dep versions for older toolchains